harness/determined/common/experimental/trial.py

```diff
@@ -185,13 +189,32 @@ def logs(
                 specific ranks.  Defaults to ``None``.
             stdtypes (List[int], optional): When set, only fetch logs from lines from the given
                 stdio outputs.  Defaults to ``None`` (same as ``["stdout", "stderr"]``).
-            min_level: (LogLevel, optional): When set, defines the minimum log priority for lines
+            min_level (LogLevel, optional): When set, defines the minimum log priority for lines
                 that will be returned.  Defaults to ``None`` (all logs returned).
+            timestamp_before (Union[str, int], optional): Specifies a timestamp that filters logs
+                before a certain time. Accepts either a string in RFC 3339 format
+                (eg. ``2021-10-26T23:17:12Z``) or an int representing the epoch second.
+            timestamp_after (Union[str, int], optional): Specifies a timestamp that filters logs
+                after a certain time. Accepts either a string in RFC 3339 format
+                (eg. ``2021-10-26T23:17:12Z``) or an int representing the epoch second.
+            sources (List[str], optional): Specifies a filter for logs based on originating node
```

Comment:
If `master` and `agent` are the only two available, should take a `StrEnum`?

Reply:
i'm hesitant to introduce new enums to the SDK that don't exist in the backend. if the backend doesn't care and just takes in strings, i don't think the SDK should either.